config/default/testRunner.conf.php

```diff
@@ -660,4 +660,10 @@
      */
     'enable-linear-next-item-warning-checkbox' => true,
 
+    /**
+     * If 'validate-response' is true, system requires a mandatory response
```

Comment:
please add a comment that this value is just default and can be overridden by a QTI test / section / item setting